.github/workflows/test_geopandas.yml

```diff
@@ -0,0 +1,48 @@
+name: Geopandas tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Folium
+      uses: actions/checkout@v4
+      with:
+        # needed to get the correct version number for Folium
+        fetch-depth: 0
+        path: folium
+
+    - name: Setup Micromamba env
+      uses: mamba-org/setup-micromamba@v2
+      with:
+        environment-name: TEST
+        create-args: >-
+          python=3
+          --file folium/requirements.txt
+
+    - name: Checkout Geopandas
+      uses: actions/checkout@v4
+      with:
+        repository: geopandas/geopandas
+        path: geopandas
+
+    - name: Install Geopandas dev dependencies
+      run: |
+        pip install -r geopandas/requirements-dev.txt
+        pip install geodatasets
```

Comment:
@martinfleis I was a bit surprised that I had to install `geodatasets` separately, is that expected? Or did I mis something? I don't mind, just checking.

Reply:
Need to update requirements-dev. Sorry for that, we're not really using it.

Reply:
No worries